apps/console/src/components/pages/protected/evidence/hooks/use-form-schema.ts

```diff
@@ -0,0 +1,36 @@
+'use client'
+import { z } from 'zod'
+import { useForm } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { addDays } from 'date-fns'
+
+const formSchema = z.object({
+  name: z.string().optional(),
+  description: z.string().min(2, {
+    message: 'Description must be at least 2 characters',
+  }),
+  tags: z.array(z.string().optional()),
+  creationDate: z.date().min(new Date(), { message: 'Start date must be in the future' }).default(new Date()),
+  renewalDate: z.date().min(new Date(), { message: 'End date must be after start date' }).default(addDays(new Date(), 365)),
```

Comment:
```suggestion
  renewalDate: z.date().min(new Date(), { message: 'Renewal date must be in the future' }).default(addDays(new Date(), 365)),
```

It can also be empty (see other comment)